lean4-infoview/src/infoview/goalCompat.ts

```diff
@@ -31,6 +31,8 @@ function transformGoalToInteractive(g: string): InteractiveGoal {
         } else if (p.match(/^([^:\n< ][^:\n⊢{[(⦃]*) :/mg)) {
             const ss = p.split(':')
             const hyp: InteractiveHypothesis = {
+                isType: false,
+                isInstance: false,
```

Comment:
```suggestion
                isType: true,
                isInstance: true,
```

We shouldn't hide anything by default (especially when there's no button to change the filtering).

Reply:
Oh then it's ok.  Got the flags backwards.

Reply:
I actually confused the filter flags with the hypothesis flags. :bulb:

Reply:
Ah but I did find another place in traceExplorer.tsx that needed a related fix.